R/clean_names.R

```diff
@@ -119,6 +120,20 @@ clean_names.tbl_graph <- function(dat, ...) {
   dplyr::rename_all(dat, .funs=make_clean_names, ...)
 }
 
+#' @rdname clean_names
+#' @export
+#' @importFrom dplyr rename_with
```

Comment:
Since you're using `dplyr::rename_with` below (referencing the package::), you shouldn't need this `@importFrom dplyr rename_with`

Reply:
Nice catch, I took it out.

Reply:
Also removed it from the tidygraph section too.